src/glog/logging.h.in

```diff
@@ -1721,17 +1741,17 @@ class GOOGLE_GLOG_DLL_DECL LogSink {
   // during this call.
   virtual void send(LogSeverity severity, const char* full_filename,
                     const char* base_filename, int line,
-                    const struct ::tm* tm_time,
-                    const char* message, size_t message_len, int32 /*usecs*/) {
+                    const LogMessageTime &logmsgtime,
+                    const char* message, size_t message_len, bool need_usec) {
     send(severity, full_filename, base_filename, line,
-         tm_time, message, message_len);
+         logmsgtime, message, message_len);
   }
   // This send() signature is obsolete.
   // New implementations should define this in terms of
   // the above send() method.
   virtual void send(LogSeverity severity, const char* full_filename,
```

Comment:
@sergiud , Is this virtual method needed? I don't think it's useful anymore. Can we remove this??

Reply:
I would leave as is for now. You can open a new PR if you wish to clean up the interface.